ixdiagnose/plugins/zfs.py

```diff
@@ -31,7 +31,13 @@ def zfs_getacl_impl(dataset_name: str, props_dict: dict) -> str:
 
 
 def resource_output(client: MiddlewareClient, resource_type: str) -> str:
-    cp = run([resource_type, 'get', 'all'], check=False)
+    if resource_type == 'zfs':
+        cp = run(
+            'zfs list -t filesystem | { count=0; while read dataset _; do [ $count -eq 0 ] && zfs get all '
```

Comment:
Why so complicated? `zfs get all -t filesystem` will return the same information excluding the snapshots.

Reply:
Should have known that too was of course possible :D